.golangci.yml

```diff
@@ -60,23 +63,31 @@ linters:
     sloglint:
       context: scope
     govet:
```

Comment:
Switching the the official golangci-lint github action caused `golangci-lint config verify` to run, which cause some issues here I had to fix